fpdf/fpdf.py

```diff
@@ -3430,7 +3478,7 @@ def multi_cell(
 
         Args:
             w (float): cell width. If 0, they extend up to the right margin of the page.
-            h (float): cell height. Default value: None, meaning to use the current font size.
+            h (float): row height. Default value: None, meaning to use the current font size.
```

Comment:
There is no notion of row there. I think `cell` is the right term

Reply:
Cell for me would mean the total height of the cell, but h controls the height of a single line of text.
The height of the cell is h times the number of text-lines.

suggestion: h (float): height of a single line of text

Reply:
Yes, your suggestion is great!